core/object/object.cpp

```diff
@@ -1739,7 +1739,7 @@ void *Object::get_instance_binding(void *p_token, const GDExtensionInstanceBindi
 			break;
 		}
 	}
-	if (unlikely(!binding)) {
+	if (unlikely(!binding && p_callbacks != nullptr)) {
```

Comment:
This can be written as `if (unlikely(!binding && p_callbacks))`

Reply:
Makes sense - that would match how `binding` is being checked!